platform/packages/versioning/src/release.rs

```diff
@@ -12,6 +12,12 @@ use super::{Version, VersionSegment};
 #[serde(transparent)]
 pub struct ReleaseLabel(String);
 
+impl ReleaseLabel {
+    pub fn void() -> Self {
+        Self("void".into())
```

Comment:
please define "void" as a constant, similar to the way we do for "RELEASE_LABEL_DEV"